misc/scripts/install-local.sh

```diff
@@ -208,11 +208,11 @@ fi' | sudo tee "$SRCDIR/$name-pacstall/DEBIAN/postrm" >"/dev/null"
 ask "Do you want to view/edit the pacscript" N
 if [[ $answer -eq 1 ]]; then
 	if [[ -n $PACSTALL_EDITOR ]]; then
-		$PACSTALL_EDITOR "$PACKAGE".pacscript
+		sudo $PACSTALL_EDITOR "$PACKAGE".pacscript
 	elif [[ -n $EDITOR ]]; then
-		$EDITOR "$PACKAGE".pacscript
+		sudo $EDITOR "$PACKAGE".pacscript
```

Comment:
Why do we even need `sudo` to edit these files? The pacscripts are downloaded in `/tmp/pacstall` which is owned by the owner.